profiles/shamilkotta.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/64640025/209474929-8e286c45-3941-4990-977e-36bb78d55833.jpeg",
+  "issueId": 1597,
+  "name": "SHAMIL K",
+  "username": "shamilkotta"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```